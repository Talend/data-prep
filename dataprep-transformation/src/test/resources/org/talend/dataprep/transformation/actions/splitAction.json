{
  "actions": [
    {
      "action": "split",
      "parameters": {
        "column_id": "birth",
        "scope": "column",
        "separator": "-",
        "limit":"3"
      }
    }
  ]
}