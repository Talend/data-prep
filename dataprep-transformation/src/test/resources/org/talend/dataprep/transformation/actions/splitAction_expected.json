{
  "records": [
    {
      "0000": "02",
      "0001": "07",
      "0002": "1964",
      "birth": "02-07-1964",
      "tdpId": 1
    },
    {
      "0000": "01",
      "0001": "02",
      "0002": "1986",
      "birth": "01-02-1986",
      "tdpId": 2
    },
    {
      "0000": "05",
      "0001": "05",
      "0002": "1982",
      "birth": "05-05-1982",
      "tdpId": 3
    },
    {
      "0000": "18",
      "0001": "09",
      "0002": "1948",
      "birth": "18-09-1948",
      "tdpId": 4
    },
    {
      "0000": "27",
      "0001": "08",
      "0002": "1969",
      "birth": "27-08-1969",
      "tdpId": 5
    },
    {
      "0000": "25",
      "0001": "07",
      "0002": "1980",
      "birth": "25-07-1980",
      "tdpId": 6
    },
    {
      "0000": "02",
      "0001": "12",
      "0002": "1965",
      "birth": "02-12-1965",
      "tdpId": 7
    },
    {
      "0000": "08",
      "0001": "04",
      "0002": "1976",
      "birth": "08-04-1976",
      "tdpId": 8
    },
    {
      "0000": "29",
      "0001": "05",
      "0002": "1954",
      "birth": "29-05-1954",
      "tdpId": 9
    },
    {
      "0000": "15",
      "0001": "10",
      "0002": "1955",
      "birth": "15-10-1955",
      "tdpId": 10
    },
    {
      "0000": "15",
      "0001": "02",
      "0002": "1962",
      "birth": "15-02-1962",
      "tdpId": 11
    },
    {
      "0000": "12",
      "0001": "01",
      "0002": "1979",
      "birth": "12-01-1979",
      "tdpId": 12
    },
    {
      "0000": "10",
      "0001": "09",
      "0002": "1974",
      "birth": "10-09-1974",
      "tdpId": 13
    },
    {
      "0000": "26",
      "0001": "09",
      "0002": "1956",
      "birth": "26-09-1956",
      "tdpId": 14
    },
    {
      "0000": "06",
      "0001": "02",
      "0002": "1987",
      "birth": "06-02-1987",
      "tdpId": 15
    },
    {
      "0000": "09",
      "0001": "06",
      "0002": "1954",
      "birth": "09-06-1954",
      "tdpId": 16
    },
    {
      "0000": "21",
      "0001": "10",
      "0002": "1955",
      "birth": "21-10-1955",
      "tdpId": 17
    },
    {
      "0000": "27",
      "0001": "07",
      "0002": "1949",
      "birth": "27-07-1949",
      "tdpId": 18
    },
    {
      "0000": "23",
      "0001": "06",
      "0002": "1987",
      "birth": "23-06-1987",
      "tdpId": 19
    },
    {
      "0000": "02",
      "0001": "04",
      "0002": "1972",
      "birth": "02-04-1972",
      "tdpId": 20
    },
    {
      "0000": "03",
      "0001": "01",
      "0002": "1953",
      "birth": "03-01-1953",
      "tdpId": 21
    },
    {
      "0000": "26",
      "0001": "09",
      "0002": "1980",
      "birth": "26-09-1980",
      "tdpId": 22
    },
    {
      "0000": "10",
      "0001": "12",
      "0002": "1975",
      "birth": "10-12-1975",
      "tdpId": 23
    },
    {
      "0000": "16",
      "0001": "01",
      "0002": "1959",
      "birth": "16-01-1959",
      "tdpId": 24
    },
    {
      "0000": "29",
      "0001": "07",
      "0002": "1954",
      "birth": "29-07-1954",
      "tdpId": 25
    },
    {
      "0000": "29",
      "0001": "12",
      "0002": "1964",
      "birth": "29-12-1964",
      "tdpId": 26
    },
    {
      "0000": "23",
      "0001": "09",
      "0002": "1954",
      "birth": "23-09-1954",
      "tdpId": 27
    },
    {
      "0000": "27",
      "0001": "10",
      "0002": "1966",
      "birth": "27-10-1966",
      "tdpId": 28
    },
    {
      "0000": "14",
      "0001": "06",
      "0002": "1948",
      "birth": "14-06-1948",
      "tdpId": 29
    },
    {
      "0000": "18",
      "0001": "01",
      "0002": "1982",
      "birth": "18-01-1982",
      "tdpId": 30
    }
  ],
  "metadata": {
    "columns": [
      {
        "name": "birth",
        "headerSize": 0,
        "type": "date",
        "quality": {
          "empty": 0,
          "invalid": 0,
          "valid": 30
        },
        "id": "birth",
        "domain": "",
        "domainLabel": ""
      },
      {
        "name": "birth_split",
        "headerSize": 0,
        "type": "integer",
        "quality": {
          "empty": 0,
          "invalid": 0,
          "valid": 30
        },
        "id": "0000",
        "domain": "",
        "domainLabel": ""
      },
      {
        "name": "birth_split",
        "headerSize": 0,
        "type": "integer",
        "quality": {
          "empty": 0,
          "invalid": 0,
          "valid": 30
        },
        "id": "0001",
        "domain": "",
        "domainLabel": ""
      },
      {
        "name": "birth_split",
        "headerSize": 0,
        "type": "integer",
        "quality": {
          "empty": 0,
          "invalid": 0,
          "valid": 30
        },
        "id": "0002",
        "domain": "",
        "domainLabel": ""
      }
    ]
  }
}
